src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewSplitAddressProcessor.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.ons.census.fwmt.outcomeservice.converter.impl;
```

Comment:
Go through the spec again
https://collaborate2.ons.gov.uk/confluence/display/CATD/FWMT+Interfaces+-+Outcomes+-+RM+-+New+Split+Address

Theres bits missing like a new Id is needed

Reply:
I am confused? we generate a new ID when we pick up the message from preprocess and convert to superset